shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -449,11 +449,11 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
   [accessibility_channel_.get() setMessageHandler:^(id message, FlutterReply reply) {
     HandleEvent((NSDictionary*)message);
   }];
-  _previous_route_ = nil;
 }
 
 AccessibilityBridge::~AccessibilityBridge() {
   view_.accessibilityElements = nil;
+  [previous_route_ release];
```

Comment:
If you just release and not `nil` it out, it will continue holding onto a dangling pointer (or worse, a live pointer with a random lifecycle).